SecondSight/Calibration.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+
+# mostly coppied from https://github.com/kyle-bersani/opencv-examples/blob/master/CalibrationByCharucoBoard/CalibrateCamera.py
```

Comment:
We need to treat this file different than the rest of the code due to being MIT licensed. We have to include the license notice from the original project

https://github.com/kyle-bersani/opencv-examples/blob/master/LICENSE

Keeping the URL there is also good
Copy and paste that license file into the top of this one

Reply:
To be clear, that comment is from the original file in DEATHSTARE++, and most of the code has been changed a lot.

Reply:
It's a very complicated topic actually. I'm happy to discuss it on Thursday. This is the safest option for sure as it's very likely the code is still covered under the MIT license